java/meterpreter/stdapi/src/main/java/com/metasploit/meterpreter/stdapi/stdapi_sys_process_getpid_V1_9.java

```diff
@@ -0,0 +1,36 @@
+package com.metasploit.meterpreter.stdapi;
+
+import com.metasploit.meterpreter.Meterpreter;
+import com.metasploit.meterpreter.TLVPacket;
+import com.metasploit.meterpreter.TLVType;
+import com.metasploit.meterpreter.command.Command;
+
+import java.lang.reflect.Method;
+
+public class stdapi_sys_process_getpid_V1_9 implements Command {
```

Comment:
This should extend stdapi_sys_process_getpid_V1_5, and re-use the classExists function